readme/manual-setup.md

```diff
@@ -64,8 +64,26 @@ $ export DISTRO_PATH=/your/path/to/ozone-distro/target/ozone-distro-1.0.0-SNAPSH
 ```
 
 ### Start Ozone
+<table>
+<tr>
+<td> macOS </td> <td> Linux </td>
+</tr>
+<tr>
+<td>
```

Comment:
It's a great suggestion to add a table for Mac and Linux os, should we also think of the Window os?